# Torsdagsopgaver_13_04_23

Endte med først at gå i gang ved en 20.30-tiden og brugte lidt lang tid på opgave 1 (har ikke lavet den helt som jeg tror I tænkte), 
så har sprunget opgave 2 over. Jeg har til gengæld brugt en Try and Catch i opgave 1 i metoden changeVolume() til at kaste en Exception (den kunne garanteret
godt være mere specifik den Exception jeg har kaldt).

Mine svar på de sidste spørgsmål i opgave 3 har jeg givet i klassen TestEdible. De står som kommentarer øverst. 

